includes/gateways/stripe/includes/admin/admin-helpers.php

```diff
@@ -166,24 +101,31 @@ function give_stripe_connect_button() {
 /**
  * Stripe Disconnect URL.
  *
+ * @param string $account_id   Stripe Account ID.
+ * @param string $account_name Stripe Account Name.
+ *
  * @since 2.5.0
  *
- * @return void
+ * @return string
  */
-function give_stripe_disconnect_url() {
+function give_stripe_disconnect_url( $account_id = '', $account_name = '' ) {
+	$args = [
```

Comment:
@mehul0810 I did not notice any nonce in this URL. Can we tighten security here?